globals.go

```diff
@@ -320,6 +320,10 @@ const (
 
 	DefaultShutdownWaitSeconds = 10
 
+	// HeartbeatTimeoutThreshold is the default timeout for how long a task can run without sending
```

Comment:
Remove "default" since it's not a configurable value - the app server will system-fail a task that exceeds this.